companies/crifbuergel.json

```diff
@@ -4,15 +4,15 @@
         "credit agency",
```

Comment:
I am unsure whether I should also change the file name/slug or that should stay permanent/fixed (for links or so?)

Reply:
I think we should keep the existing slug, otherwise links for existing requests in "My requests" will break. Somewhat related discussion: #1157.